build/releaseBuild.yml

```diff
@@ -202,6 +202,14 @@ jobs:
       BuildDropPath: '$(Build.SourcesDirectory)\src'
       ManifestDirPath: '$(Build.SourcesDirectory)\artifacts'
 
+  - task: NuGetCommand@2
```

Comment:
do we need this? @jmprieur

Reply:
This would be great to put behind a var

Reply:
Re-added a condition that was previously removed

Reply:
> do we need this? @jmprieur

I don't think so if we want to be intentional about the previews.
If we keep it (I'm fine with that for the nightly), then we absolutely need to guard it with a condition:

```
   condition: eq(variables['NugetPackageType'], 'preview')
```

Reply:
Oh and it's not up to date with the content of dev. It won't even build any longer as the sccript don't exist.

Reply:
so, let's remove it. you'd have a merge conflict anyways, @sruke

Reply:
it should be removed. no longer needed

Reply:
Had an internal discussion and decided to keep the task. A condition was added as a gate.